icons/sparkles.json

```diff
@@ -5,16 +5,28 @@
     "effect",
     "filter",
     "night",
-    "magic"
+    "magic",
+    "dark",
+    "sky",
+    "twinkle",
+    "shine",
+    "planet",
+    "space",
+    "galaxy",
+    "universe",
+    "astronomy",
+    "astrology",
+    "astrophysics"
```

Comment:
We probably can just use the word `astro` here. And it will show on all these words